kobo-uncaged/kunc/kunc.go

```diff
@@ -44,26 +42,31 @@ func New(kobo *device.Kobo) *koboUncaged {
 	return &koboUncaged{kobo}
 }
 
+func (ku *koboUncaged) SelectCalibreInstance(calInstances []uc.CalInstance) uc.CalInstance {
+	return calInstances[0]
```

Comment:
Are you sure the list will never be empty?

Reply:
From memory, UNCaGED will error out if no Calibre instances are found, but I'll double check that.

Reply:
Yep, just checked, UNCaGED does the test.

```
if len(c.calibreInstances) == 0 {
		return nil, fmt.Errorf("New: Could not find calibre instance: %w", CalibreNotFound)
	}
	c.calibreAddr = c.client.SelectCalibreInstance(c.calibreInstances).Addr
```